backend/src/authentication/session.rs

```diff
@@ -43,6 +43,7 @@ impl Session {
     }
 
     /// Get the session key
+    #[cfg(test)]
```

Comment:
This is not a test right?

Reply:
`#[cfg(test)]` does not mean its a test, the code should only below `#[cfg(test)]` should only be included when running `cargo test`. In this case this method is not used anywhere, except for in tests. In the future that might change.

Reply:
Aaahh I had no idea, thanks for enlightening me :smile: